iModelCore/ECPresentation/Tests/NonPublished/Helpers/TestHelpers.cpp

```diff
@@ -24,6 +36,8 @@ void RulesEngineTestHelpers::InitSchemaRegistry(ECDbR ecdb, bvector<bpair<Utf8St
             BeAssert(false);
             continue;
             }
+        if (testNamesThatShouldRun.find(pair.first) == testNamesThatShouldRun.end())

```

Comment:
Shouldn't this check be done before reading schema xml?

Reply:
Fixed